### DISCUSSION TITLE
Notifications for Birds Not on Life List?

### DISCUSSION THREAD

**Discussion by enthusiastpodcast** [mcguirepr89/BirdNET-Pi#1184] [Q&A]
Got my BirdNET-Pi set up and running, and I love it so far 🤩 I don't have notifications enabled, as that would be an obscene amount of info pushed to my phone. However, something that would be great would be enabling notifications if any bird not on my eBird Life List was detected — is there any way to achieve something like this? I know I could limit what birds are _detected_, but I don't want to mess with that. TIA!

**Comment by alexbelgium** [comment]
Hi, i'm using the option to send notification only for rare birds. That way a notification is sent only if it was detected less than 5 times in the past week, which is already great to target misidentifications (no, alas I don't have rare exotic birds but I do have overlapping songs of common birds while raining) or more excitingly new specie is that you can double check using the recording! 

BTW I'm an enthusiast having recently discovered birdnet-pi but there is a more recent and up-to-date fork here : https://github.com/Nachtzuster/BirdNET-Pi
